### PR TITLE
Implemented '/stories' API endpoints

### PR DESCRIPTION
**Pull request by kinsomicrote**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ doc
 pkg
 spec/reports
 tmp
+.idea
+.rakeTasks
```
Comment (christopherstyles) [inline .gitignore]: These seem like application-specific/custom settings? These should probably go in your local system’s gitignore instead of here.

Comment (kinsomicrote) [reply .gitignore]: Alright. I'll fix up and push

Comment (kinsomicrote) [reply .gitignore]: @christopherstyles PR Updated.

File: lib/ngp_van/client/stories.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module NgpVan
+  class Client
+    module Stories
```
[the remark this answers is not in the document]
Comment (christopherstyles) [reply lib/ngp_van/client/stories.rb]: I disabled this cop, please ignore.

File: spec/ngp_van/client/stories_spec.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module NgpVan
+  class Client
+    RSpec.describe Stories do
```
[the remark this answers is not in the document]
Comment (christopherstyles) [reply spec/ngp_van/client/stories_spec.rb]: I disabled this cop, please ignore.

**Comment by christopherstyles** [comment]
@kinsomicrote I’ve recently added Hound to this repo, so it’s reporting a few style issues if you wouldn’t mind fixing those up (we use rubocop, and it can probably auto-fix most of those for you with the `--auto-correct` option).

**Comment by kinsomicrote** [comment]
@christopherstyles I think I have resolved all issues. Let me know if there's something I missed.
Thanks.

File: spec/ngp_van/client/stories_spec.rb [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      let(:client) { NgpVan::Client.new }
+
+      describe '#create_stories' do
```
Comment (christopherstyles) [inline spec/ngp_van/client/stories_spec.rb]: @kinsomicrote Would you mind singularizing this?
```suggestion
      describe '#create_story' do
```

Comment (kinsomicrote) [reply spec/ngp_van/client/stories_spec.rb]: @christopherstyles done

**Comment by christopherstyles** [comment]
Excellent, thanks so much @kinsomicrote!

**Pull request merged**